pkg/hub/hubclient/hubclient.go

```diff
@@ -129,7 +141,16 @@ func (hubClient *HubClientConfig) UpdateNodePortForSliceGwServer(ctx context.Con
 
 	sliceGw.Spec.LocalGatewayConfig.NodePort = int(sliceGwNodePort)
 
-	return hubClient.Update(ctx, sliceGw)
+	err = hubClient.Update(ctx, sliceGw)
+	hubClient.eventRecorder.RecordEvent(ctx, &monitoring.Event{
+		EventType:         monitoring.EventTypeNormal,
+		Reason:            monitoring.EventReasonNodePortUpdate,
+		Message:           fmt.Sprintf("NodePort Updated to: %d", sliceGwNodePort),
+		ReportingInstance: "Controller Reconciler",
+		Object:            sliceGw,
+	})
```

Comment:
Here we do not report "Action" field but, we do this in the case of NodeIpUpdate above.

Reply:
nice find. fixed